sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,38 +1,37 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (20/11/10)
 
 ### Breaking Changes
```

Comment:
Renaming an exception type - breaking change:
https://github.com/Azure/azure-sdk-for-cpp/pull/825

Move from "New feature" to breaking change.

Reply:
Another breaking change:
https://github.com/Azure/azure-sdk-for-cpp/pull/833

Reply:
Public header change - breaking?
https://github.com/Azure/azure-sdk-for-cpp/pull/907

Reply:
> #825 is in breaking change already: - Calling Cancel() from context now throws OperationCanceledException.

Why is that in beta.2 and not beta.3? Also, this only mentions that we are changing the exception being thrown. We need to mention that we removed or renamed `RequestCanceledException` as part of the breaking change as well.